src/event/format/mod.rs

```diff
@@ -204,6 +204,24 @@ pub fn override_timestamp_fields(
     Arc::new(Schema::new(updated_fields))
 }
 
+/// All number fields from inferred schema are forced into Float64
+pub fn override_num_fields_from_schema(schema: Vec<Arc<Field>>) -> Vec<Arc<Field>> {
+    schema
+        .iter()
+        .map(|field| {
+            if field.data_type().is_numeric() && field.data_type() != &DataType::Float64 {
```

Comment:
```suggestion
            if field.data_type().is_numeric() {
```

Effect is the same, slight cost benefit from the other, but higher mental load to read

Reply:
This is just a suggestion, we can make do with a note comment atop this

Reply:
This is just a suggestion, we can make do with a note comment atop this